person.rb

```diff
@@ -1,15 +1,20 @@
 require_relative 'corrector'
+require_relative 'book'
+require_relative 'rental'
+require_relative 'student'
+require_relative 'teacher'
 
 class Person
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] There should be getter and setter (attr_accessor) for **:rentals**. Kindly add this.